src/languages/java.js

```diff
@@ -15,6 +15,17 @@ function(hljs) {
     'package default double public try this switch continue throws protected public private ' +
     'module requires exports do';
 
+  var ANNOTATION = {
+    className: 'meta',
+    begin: '@[A-Za-z0-9]+',
```

Comment:
Shouldn't it be @ + JAVA_IDENT_RE?..

Reply:
I don't know Java well enough to say for sure.  If you're confident I'll change it, if now lets leave it be.

Reply:
Well, let see... https://docs.oracle.com/javase/specs/jls/se7/html/jls-18.html
```
Annotation:
    @ QualifiedIdentifier [ ( [AnnotationElement] ) ]
```
So it's just a regular Java identifier.